js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -82,7 +82,7 @@ const TickDetailsDialogImpl = ({tickId, tickResultType, instigationSelector}: In
   const [activeTab, setActiveTab] = useState<'result' | 'logs'>('result');
 
   const {data} = useQuery<SelectedTickQuery, SelectedTickQueryVariables>(JOB_SELECTED_TICK_QUERY, {
-    variables: {instigationSelector, tickId: tickId || 0},
+    variables: {instigationSelector, tickId: tickId || ''},
```

Comment:
both the old code and new code here seem sketchy, should we short-circuit and not run the query at all when its undefined?

Reply:
I couldn't quite figure out the react hook wrangling to make that work

Reply:
The line below (`skip: !tickId`) should already do that. This `||` is probably just to appease TypeScript.